### ISSUE TITLE
How do I set a goal column in Evil?

### ISSUE THREAD

**Issue by LemonBreezes** [emacs-evil/evil#1831]
# Issue type
Question

Hi. I'm just wondering if there is an analogue of `set-goal-column` for Evil for column editing?

**Comment by axelf4** [comment]
No, but evil-next-/previous-line already respect `goal-column`, and there are other more "Vim-iomatic" ways of doing column editing. Did you have a specific use case in mind?

**Comment by LemonBreezes** [comment]
> No, but evil-next-/previous-line already respect `goal-column`, and there are other more "Vim-iomatic" ways of doing column editing. Did you have a specific use case in mind?

Yeah, I wanted to insert numbers for music tracks in Dired and what came to mind was using `kmacro` and `goal-column` to delete the numbers that were already there but `C-x C-n` is bound in insert state so I got the idea that Vim users never needed that command.
